alkira/resource_alkira_policy_nat_rule.go

```diff
@@ -158,6 +165,13 @@ func resourceAlkiraPolicyNatRule() *schema.Resource {
 							Type:        schema.TypeBool,
 							Optional:    true,
 						},
+						"egress_type": {
+							Description:  "The egress type to use with the match. Options are are `ALKIRA_PUBLIC_IP` or `BYOIP`.",
```

Comment:
If Default value is `NONE`, then the possible options are `ALKIRA_PUBLIC_IP`, `BYOIP` and `NONE`, right?

Reply:
No, I copied the field definition structure and missed updating the default value. There is no default value. Will update